.github/workflows/main.yml

```diff
@@ -28,12 +28,12 @@ jobs:
           files: ./coverage/script/coverage-final.json
           flags: script
           name: codecov-script
-          platform: ${{ os.platform }}
+          platform: ${{ matrix.platform }}
```

Comment:
`matrix.os` or `matrix.include`, perhaps?

Reply:
I think it was `matrix.platform` that ended up working here